src/modules/Storage.rb

```diff
@@ -354,6 +354,9 @@ def InitLibstorage(readonly)
       @conts = getContainers
       log.info("InitLibstorage conts:#{@conts}")
 
+      # Initializes default subvolume name (bsc#1040154)
+      FileSystems.read_default_subvol_from_target if Mode.normal || Mode.config
```

Comment:
Um, and what about the "firstboot" mode? How it should behave in that mode?

(I'm not sure how much the storage module would make sense in the firstboot mode, but anyway...)